src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -129,10 +129,11 @@ public void allowRenewalWithHoldsWhenProfileIsRollingUseLoanPeriod() throws Exce
       .unlimitedRenewals()
       .renewFromSystemDate();
 
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
```

Comment:
This is repeated 11 times. Each one should be replaced by `use(rollingPolicy)` or `use(fixedPolicy)` or ...

Reply:
Done.